changelogs/fragments/5539-adding-features-to-ilo_redfish_utils.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - added new features for ilo_redfish_utils (https://github.com/ansible-collections/community.general/pull/5539/).
```

```suggestion
  - ilo_redfish_utils module utils - added new features (https://github.com/ansible-collections/community.general/pull/5539/).
```
Would probably be nice to be a little bit more concrete, about what these features are for.